.circleci/config.yml

```diff
@@ -75,7 +75,7 @@ workflows:
       - release-management/release-package:
           sign: true
           github-release: true
-          node_version: lts
+          node_version: '14'
```

Comment:
Beware of using node v14 on windows.
We pinned v16 and v17 to known safe versions here: https://github.com/forcedotcom/npm-release-management-orb/pull/93

v14 was ok then but between that pr and today maybe a new 14 with the bad npm was released and jobs will not run.